### DISCUSSION TITLE
Volunteer- Meeting Minutes 05/19/2022

### DISCUSSION THREAD

**Discussion by Ritakang0451** [WeBankFinTech/WeDataSphere#32] [General]
#### Date 日期：05月/ 19 日 /2022年

- Place: online 

- Members in attendance 出席人员
  @jwangkun[jwangkun](https://github.com/orgs/WeBankFinTech/people/jwangkun) 
	@brucedchen1991	[Peace chen](https://github.com/orgs/WeBankFinTech/people/brucedchen1991)  
	@pengge00[pengge00](https://github.com/orgs/WeBankFinTech/people/pengge00)
	@Ritakang0451[康月牙ritakang](https://github.com/orgs/WeBankFinTech/people/Ritakang0451)  
	@yangli-os[YangLi](https://github.com/orgs/WeBankFinTech/people/yangli-os)  
	@Clairedenggg[Yongjia Teng](https://github.com/orgs/WeBankFinTech/people/Clairedenggg) 
	@Yuhang-Luky[Yuhang-Luky](https://github.com/orgs/WeBankFinTech/people/Yuhang-Luky) 
@董鹏

- Scheduled staring time 开始时间:  20:00-21:00
 
- Discussion topic讨论话题: 
1、工作组会议周期
2、工作看板工具
3、事项设置&认领 
 
- Bullet points重点记录 
1、Apache linkis 邮件群组运营：5月@rita负责，李扬共同参与梳理SOP ，6月@李扬@luky负责
2、Apache linkis web 新闻模块：5月@peace，计划5月29日（星期日）做内容更新
3、Apache linkis 安装视频：6月17日（星期五）@John输出
4、meetup：5月28日（星期六）社区活动易拉宝设计@Clairedenggg，5月26（星期四）输出制作，@Rita5月20（星期五）提供内容
5、GitHub issue 处理：@pengge00 怎么规范化issue处理方式

- Resolution （针对上述讨论话题）通过的决议
1、会议周期：每两周一次会议，预计时长1 h 
2、事项设置&认领：要更具体，标准化，有明确的时间节点
3、扩展性内容：过程中总结，发现，迭代
4、看板：暂时延用GitHub作为看板， 其它可调研语雀，腾讯文档，GitHub，JIRA ，

- Date of next meeting下次会议时间: 每两周，6月第一周,下次会议主持@李扬

**Comment by Yuhang-Luky** [comment]
感谢您的来信，小卢同学已收到！
